privacy/enforcer_test.go

```diff
@@ -284,83 +282,90 @@ func TestAllowComponentEnforcementRule(t *testing.T) {
 		{
 			name: "activity_is_allowed_componentType_only",
 			componentRule: ComponentEnforcementRule{
-				allowed:       true,
+				result:        ActivityAllow,
 				componentType: []string{"bidder"},
 			},
 			target:         ScopedName{Scope: "bidder", Name: "bidderB"},
 			activityResult: ActivityAllow,
 		},
 		{
-			name: "abstain_activity_no_componentType_and_no_componentName",
+			name: "no-conditions-allow",
 			componentRule: ComponentEnforcementRule{
-				allowed: true,
+				result: ActivityAllow,
 			},
-			target:         ScopedName{Scope: "bidder", Name: "bidderB"},
-			activityResult: ActivityAbstain,
+			target:         ScopedName{Scope: "bidder", Name: "bidderA"},
+			activityResult: ActivityAllow,
+		},
```

Comment:
Per the spec:

> Note that rules can have no conditions, in which case they automatically match and the "allow" value is utilized.

I don't expect to see this actually used, but this requirement simplifies the evaluation logic.

Reply:
Ok, but it doesn't really make sense. Should it be a validation error? Should we skip Evaluation to the next rule? 
Allow is always true or false which means we will always return without further checking (after first rule without conditions):
```go
for _, rule := range p.rules {
		result := rule.Evaluate(target)
		if result == ActivityDeny || result == ActivityAllow {
			return result
		}
	}
```

Reply:
> which means we will always return without further checking (after first rule without conditions)

You're right.

Per the spec, processing will end at the first rule without conditions and that "allow" value will be used. I don't expect any host to actually set a rule without conditions, but if it does happen somehow I'm cool with what the spec defines. I think we can argue either way on that behavior and this way is simpler to code.

Reply:
Ok, this works too.